api/v4/openapi.yaml

```diff
@@ -164,6 +166,10 @@ components:
           type: object
           additionalProperties:
             $ref: '#/components/schemas/Source'
+        unscanned:
+          type: array
+          items:
+            $ref: '#/components/schemas/PackageRef'
```

Comment:
Can we add here a `reason` why it has not been scanned? e.g. `unsupported-pkg-type`